tools/cbmc/proofs/Queue/QueueGenericCreate/QueueGenericCreate_harness.c

```diff
@@ -0,0 +1,19 @@
+#include "FreeRTOS.h"
+#include "queue.h"
+#include "queue_datastructure.h"
+#include "cbmc.h"
+
+
+void harness(){
+	UBaseType_t uxQueueLength;
+	UBaseType_t uxItemSize;
+	uint8_t ucQueueType;
+	__CPROVER_assume(uxQueueLength > ( UBaseType_t ) 0);
```

Comment:
What happens if the user tried to create a queue of length 0?  Would that be legal?

Reply:
This is not yet specified. The function is supposed to be called with uxQueueLength > 0.

Reply:
Is that documented in the mainline code, ideally as an assertion?